tarorpc/taro.proto

```diff
@@ -16,6 +16,12 @@ service Taro {
     */
     rpc ListAssets (ListAssetRequest) returns (ListAssetResponse);
 
+    /* tarocli: `assets utxos`
+   ListUtxos lists the UTXOs managed by the target daemon, and the assets they
+   hold.
+   */
```

Comment:
nit: indentation seems to be off by one space here.

Reply:
You're right, good eye 😅  https://github.com/lightninglabs/taro/pull/185/commits/2a194865c25d7959790172ca20803fc5392ac584